2025-02-25 12:26:15,116 INFO    MainThread:62292 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-02-25 12:26:15,116 INFO    MainThread:62292 [wandb_setup.py:_flush():67] Configure stats pid to 62292
2025-02-25 12:26:15,116 INFO    MainThread:62292 [wandb_setup.py:_flush():67] Loading settings from C:\Users\asp61\.config\wandb\settings
2025-02-25 12:26:15,117 INFO    MainThread:62292 [wandb_setup.py:_flush():67] Loading settings from f:\VS Files\Labs sem 4\MLPR\Distance-Based-Classification\wandb\settings
2025-02-25 12:26:15,117 INFO    MainThread:62292 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-02-25 12:26:15,117 ERROR   MainThread:62292 [wandb_setup.py:_flush():69] error in wandb.init()
Traceback (most recent call last):
  File "C:\Users\asp61\AppData\Roaming\Python\Python313\site-packages\IPython\core\interactiveshell.py", line 3579, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\asp61\AppData\Local\Temp\ipykernel_62292\3551603027.py", line 13, in <module>
    wandb.init(project='distance_classification_project')
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\wandb_init.py", line 1485, in init
    wandb._sentry.reraise(e)
    ~~~~~~~~~~~~~~~~~~~~~^^^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\analytics\sentry.py", line 156, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\wandb_init.py", line 1433, in init
    wi.maybe_login(init_settings)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\wandb_init.py", line 175, in maybe_login
    wandb_login._login(
    ~~~~~~~~~~~~~~~~~~^
        anonymous=run_settings.anonymous,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
        _silent=run_settings.quiet or run_settings.silent,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\wandb_login.py", line 311, in _login
    wlogin.try_save_api_key(key)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\wandb_login.py", line 176, in try_save_api_key
    apikey.write_key(self._settings, key)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\asp61\AppData\Local\Programs\Python\Python313\Lib\site-packages\wandb\sdk\lib\apikey.py", line 294, in write_key
    raise ValueError(
        "API key must be 40 characters long, yours was {}".format(len(key))
    )
ValueError: API key must be 40 characters long, yours was 1
2025-02-25 12:26:15,121 INFO    MainThread:62292 [wandb_init.py:setup_run_log_directory():647] Logging user logs to f:\VS Files\Labs sem 4\MLPR\Distance-Based-Classification\wandb\run-20250225_122615-af788szd\logs\debug.log
2025-02-25 12:26:15,121 INFO    MainThread:62292 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to f:\VS Files\Labs sem 4\MLPR\Distance-Based-Classification\wandb\run-20250225_122615-af788szd\logs\debug-internal.log
2025-02-25 12:26:15,121 INFO    MainThread:62292 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000282EDFD9810>
2025-02-25 12:26:15,121 INFO    MainThread:62292 [wandb_init.py:init():761] calling init triggers
2025-02-25 12:26:15,122 INFO    MainThread:62292 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-02-25 12:26:15,122 INFO    MainThread:62292 [wandb_init.py:init():784] starting backend
2025-02-25 12:26:15,122 INFO    MainThread:62292 [wandb_init.py:init():788] sending inform_init request
2025-02-25 12:26:15,234 INFO    MainThread:62292 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-02-25 12:26:15,236 INFO    MainThread:62292 [wandb_init.py:init():803] backend started and connected
2025-02-25 12:26:15,240 INFO    MainThread:62292 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-02-25 12:26:15,294 INFO    MainThread:62292 [wandb_init.py:init():896] updated telemetry
2025-02-25 12:26:15,379 INFO    MainThread:62292 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-02-25 12:26:20,040 INFO    MainThread:62292 [wandb_init.py:init():995] starting run threads in backend
2025-02-25 12:26:20,411 INFO    MainThread:62292 [wandb_run.py:_console_start():2377] atexit reg
2025-02-25 12:26:20,411 INFO    MainThread:62292 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-02-25 12:26:20,411 INFO    MainThread:62292 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-02-25 12:26:20,411 INFO    MainThread:62292 [wandb_run.py:_redirect():2317] Redirects installed.
2025-02-25 12:26:20,414 INFO    MainThread:62292 [wandb_init.py:init():1037] run started, returning control to user process
2025-02-25 12:26:20,416 INFO    MainThread:62292 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-25 12:26:20,416 INFO    MainThread:62292 [wandb_init.py:_pause_backend():564] pausing backend
2025-02-25 12:26:26,271 INFO    MainThread:62292 [wandb_init.py:_resume_backend():569] resuming backend
2025-02-25 12:26:27,481 INFO    MainThread:62292 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-25 12:26:27,481 INFO    MainThread:62292 [wandb_init.py:_pause_backend():564] pausing backend
